src/lib/data/buyables/perdu.ts

```diff
@@ -0,0 +1,293 @@
+import resolveItems from '../../util/resolveItems';
+import { Buyable } from './buyables';
+
+export const perduBuyables: Buyable[] = [
+	{
+		name: 'Karamja gloves 1',
+		gpCost: 1000,
+		ironmanPrice: 200,
+		collectionLogReqs: resolveItems(['Karamja gloves 1'])
+	},
```

Comment:
I think this entire file could be replaced with just:
```ts
diariesCL.map(itemName => ({ name: itemName, gpCost: 1000, ... })
```

Reply:
Resolved the above comment